src/agent/agent_info/src/agent_info_persistance.cpp

```diff
@@ -142,6 +215,11 @@ std::string AgentInfoPersistance::GetUUID() const
     return GetAgentInfoValue("uuid");
 }
 
+std::vector<std::string> AgentInfoPersistance::GetGroups() const
+{
+    return GetAgentGroupList();
```

Comment:
Why don't just get the groups in this function instead of calling another one?

Reply:
Done!